### PR TITLE
Fix comp

### PR DESCRIPTION
**Pull request by Jarkyman**
What kind of change does this PR introduce?

- [ ] `bug-fix`
- [ ] `new-feature`
- [x] `enhancement`

### CODE REVIEW

**Review by Andreassim — APPROVED** [review]
lgtm

File: src/main/java/com/app/controllers/InputController.java [hunk 1]
```diff
@@ -139,21 +138,17 @@ public static CompetitionModel validateCompetitionsId(ArrayList<CompetitionModel
...
-        if (c.getId().equals(result)) {
-          return c;
-        }
+      if (result.equalsIgnoreCase("s")) {
+        return result;
       }
-
-      VIEW.printInlineWarning("Not a valid ID. Please try again: ");
+      VIEW.printInlineWarning("Not a valid choice. Please try again: ");
+      result = IN.nextLine();
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/InputController.java:151]: Use InputController

File: src/main/java/com/app/controllers/ResultController.java [hunk 2]
```diff
@@ -30,8 +30,11 @@ public ResultModel addResultTime(MemberModel member, CompetitionModel competitio
             "00:" + InputController.validateCompetitionResultTime(),
             DateTimeFormatter.ofPattern("HH:mm:ss:SS"));
 
-    VIEW.printInline("Placement: ");
-    String placement = InputController.validatePlacement();
+    String placement = "0";
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/ResultController.java]: ?

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+    VIEW.printInline("Date [dd/MM/yyyy]: ");
+    LocalDate date =
+        LocalDate.parse(InputController.validateDate(), DateTimeFormatter.ofPattern("dd/MM/yyyy"));
+
+    VIEW.printInline("Please enter start time of the practice [HH:mm]: ");
+    LocalTime startTime =
+        LocalTime.parse(
+            InputController.validateCompetitionTime(), DateTimeFormatter.ofPattern("HH:mm"));
+
+    VIEW.printInline("Junior or senior [j/s]: ");
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java:75]: Use `VIEW.displayOptions`

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+    AgeGroupType ageGroup = findAgeGroup(InputController.validateAgeGroup());
+
+    String name = ageGroup.name() + " practice " + date + " " + startTime;
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java:78]: ?

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+        MEMBER_CONTROLLER.viewTableMembers();
+        VIEW.printInline("Member ID: ");
+        MemberModel member =
+            getMember(InputController.validateMemberId(new MemberController().getMembers()));
+        do {
+          addResultToCompetition(competition, resultController.addResultTime(member, competition));
+          VIEW.printInline("Do you wish to add another result to this member [Y/N]: ");
+        } while (InputController.promptYesNo());
+        VIEW.printInline(
+            "Do you wish to add results for another member, on this competition [Y/N]: ");
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java]:
```suggestion
            "Do you wish to add results for another member, on this competition [Y/n]: ");
```

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+      SwimEventModel competition = InputController.validateSwimEventId(swimEventModels);
+
+      do {
+        MEMBER_CONTROLLER.viewTableMembers();
+        VIEW.printInline("Member ID: ");
+        MemberModel member =
+            getMember(InputController.validateMemberId(new MemberController().getMembers()));
+        do {
+          addResultToCompetition(competition, resultController.addResultTime(member, competition));
+          VIEW.printInline("Do you wish to add another result to this member [Y/N]: ");
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java]:
```suggestion
          VIEW.printInline("Do you wish to add another result to this member [Y/n]: ");
```

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 7]
```diff
@@ -0,0 +1,339 @@
...
+      SwimEventModel practice = InputController.validateSwimEventId(swimEventModels);
+
+      do {
+        MEMBER_CONTROLLER.viewTableMembers();
+        VIEW.printInline("Member ID: ");
+        MemberModel member =
+            getMember(InputController.validateMemberId(new MemberController().getMembers()));
+        do {
+          addResultToCompetition(practice, resultController.addResultTime(member, practice));
+          VIEW.printInline("Do you wish to add another result to this member [Y/N]: ");
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java]:
```suggestion
          VIEW.printInline("Do you wish to add another result to this member [Y/n]: ");
```

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 8]
```diff
@@ -0,0 +1,339 @@
...
+        } while (InputController.promptYesNo());
+        VIEW.printInline("Do you wish to add results for another member, on this practice [Y/N]: ");
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java]:
```suggestion
        VIEW.printInline("Do you wish to add results for another member, on this practice [Y/n]: ");
```

File: src/main/java/com/app/controllers/SwimEventController.java [hunk 9]
```diff
@@ -0,0 +1,339 @@
...
+  }
+
+  /**
+   * Converts and array of result models to a 2d String array
+   *
+   * @param resultTimes Arraylist of result times, that needs to be converted to a string
+   * @return A 2d String array
+   */
+  private String[][] arrayWithResultToDisplay(ArrayList<ResultModel> resultTimes) {
+    String[][] result = new String[resultTimes.size()][5];
```
Comment (MathiasReker) [inline src/main/java/com/app/controllers/SwimEventController.java]: Softcode 5

**Pull request merged**